src/plugins/intel_gpu/src/graph/impls/ocl_v2/group_normalization_bfyx_opt.hpp

```diff
@@ -48,6 +48,17 @@ struct GroupNormalizationBfyxOpt : public ImplementationManager {
 
         return true;
     }
+    [[nodiscard]] in_out_fmts_t query_formats(const program_node& node) const override {
+        std::vector<format::type> in_fmts(node.get_dependencies().size(), format::any);
+        std::vector<format::type> out_fmts(node.get_outputs_count(), format::any);
+
+        for (size_t i = 1; i < node.get_dependencies().size(); i++) {
+            size_t in_rank = node.get_input_layout(i).get_rank();
+            in_fmts[i] = format::get_default_format(in_rank);
+        }
+
+        return {in_fmts, out_fmts};
+    }
```

Comment:
Please move this function to its parent, and re-use it in all occurrences.

Reply:
Move query_formats to GN base class as you commented. Thanks for your comments.